docs/src/CommutativeAlgebra/GroebnerBases/orderings.md

```diff
@@ -159,14 +159,24 @@ $x^\alpha > x^\beta \;  \Leftrightarrow \;  \deg(x^\alpha) > \deg(x^\beta)  \;\t
 deglex(R::MPolyRing)
 ```
 
-#### The Reverse Lexicographical Ordering
+#### The Inverse Lexicographical Ordering
 
-The *reverse lexicographical ordering* `revlex` is defined by setting
+The *inverse lexicographical ordering* `invlex` is defined by setting
 
 $x^\alpha > x^\beta \;  \Leftrightarrow \;\exists \; 1 \leq i \leq n: \alpha_n = \beta_n, \dots, \alpha_{i+1} = \beta_{i+1}, \alpha_i  > \beta_i.$
 
 ```@docs
-revlex(R::MPolyRing)
+invlex(R::MPolyRing)
+```
+
+#### The Degree Inverse Lexicographical Ordering
+
+The *degree inverse lexicographical ordering* `deginvlex` is defined by setting
+
+$x^\alpha > x^\beta \;  \Leftrightarrow \deg(x^\alpha) > \deg(x^\beta)  \;\text{ or } \;\exists \; 1 \leq i \leq n: \alpha_n = \beta_n, \dots, \alpha_{i+1} = \beta_{i+1}, \alpha_i  > \beta_i.$
```

Comment:
Not a change request for this PR, but perhaps for a future round of improvements:

I think we should move all these definitions into the docstrings, so that they are also available in the REPL. (Alternatively: leave these nicely LaTeX formatted defs here, and use a simplified version better suited for viewing in the REPL in the docstrings).

Reply:
Will discuss this with @wdecker  and handle it in a different PR.